src/arrays.ts

```diff
@@ -72,11 +72,16 @@ export const unique = <T>(
         .map((sameItems: ReadonlyArray<T>) => sameItems.reduce(mergeFn));
 };
 
-export const arrayEquals = <T>(a: T[], b: T[], cmp: (a: T, b: T) => boolean = (x: T, y: T) => x === y): boolean => {
+export const arrayEquals = <T>(
+    a: readonly T[],
+    b: readonly T[],
+    // tslint:disable-next-line: strict-comparisons
+    cmp: (a: T, b: T) => boolean = defaultCompareFunction,
```

Comment:
Why disable this rule here when it has already been disabled for defaultCompareFunction?

Reply:
I forgot to remove that line again, after I made it use the `defaultCompareFunction` :smile: